changelog/fix-6941-warning-for-negative-balance

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: fix
+Comment: Negative balance notice will only show when total balance is below 0.
```

Comment:
I think it would be clearer to name the bug head on. E.g. something like "Fixed a bug where the 'deposits paused while balance is negative' notice was erroneously shown after an instant deposit".

Reply:
Updated.